alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/pom.xml

```diff
@@ -91,6 +91,11 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <LIBREOFFICE_HOME>/opt/libreoffice6.3</LIBREOFFICE_HOME>
```

Comment:
Should we configure 7.0.7 as a top-level pom version property to help we catch references (at least poms & Dockerfiles) .... we would still need to remember to also update defaults in ootb YAML files.

Reply:
These property variables are only used in tests as they are part of the surefire-plugin.

Reply:
OK, that's fine - we need to remember to change in all places when we next upgrade LibreOffice version.